theme/management/commands/migrate_quota_to_minio.py

```diff
@@ -19,10 +19,13 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         for quota in UserQuota.objects.all():
-            try:
-                subprocess.run(["mc", "quota", "set", f"{quota.zone}/{quota.user.userprofile.bucket_name}",
-                                "--size", f"{quota.allocated_value}{_convert_unit(quota.unit)}"],
-                               check=True,
-                               )
-            except subprocess.CalledProcessError as e:
-                raise ValidationError(f"Error setting quota: {e}")
+            if quota.user.is_active:
```

Comment:
Any specific reason that we don't want quota set for inactive users?

Reply:
Active user with resources is the criteria used when we created buckets. https://github.com/hydroshare/hydroshare/blob/develop/hs_core/management/commands/create_buckets.py#L33

Looking at this, we could tighten up the criteria a bit by checking if the user has a bucket instead of is_active.

Reply:
makes sense, thanks